src/pelicanfs/core.py

```diff
@@ -515,47 +542,50 @@ async def _glob(self, path, maxdepth=None, **kwargs):
                 p.rstrip("/")
                 if not append_slash_to_dirname
                 and info["type"] == "directory"
-                and p.endswith("/")
```

Comment:
What prompted the reformatting here? I find the original more readable, and I think it's what `black` produces.

Reply:
Not sure. I think one of the plugins reformatted it. I can try adding black, but I think I got a bit too many issues with it.

Reply:
Not sure, one of the plugins did it. I can try using black and hopefully it won't fight with flake8 again.

Reply:
Ah yes, I know. Because the line-length is set so long (and no, I'm not going to change that), black takes that to mean that anything below that which can be one line, should be. So it changes nicer formatted stuff to one ugly line. 

I tried to get it back to the original after removing black and it failed. I added black back in and you can see the results.

Since it's just changing line length things and I'm trying my best to ignore line-length limitation (and stop it from enforcing it in the revers), I figured removing black was the better option.